azure-pipelines.yml

```diff
@@ -209,31 +209,31 @@ steps:
       package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
     displayName: 'Deploy to cloudroll'
     
-  # TODO - delete after release pipeline creation
```

Comment:
We talked about this briefly yesterday in the sync. After seeing this PR in more detail, I think the changes in this file (azure-pipelines.yml) should be moved into a separate PR please.